Corgibytes.Freshli.Lib/ScanResult.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Corgibytes.Freshli.Lib
+{
+    public record ScanResult(string Filename, List<MetricsResult> MetricResults)
+    {
+        public override string ToString()
```

Comment:
I don't think there are any tests for the `ToString` method.  Can you add one or two tests please and thank you.